.eslintrc.js

```diff
@@ -44,5 +44,6 @@ module.exports = {
         "no-shadow": "error",
         "no-useless-concat": "error",
         "no-undef": "off",
+        "prettier/prettier": "error",
```

Comment:
Can we make this a `"prettier/prettier": "warn"` and move it as the first rule?

Reply:
What is the reasoning for `warn`? `error` will enforce style during CI, so all new code is guaranteed to be consistent.

Reply:
Hmm, I see. I just don't know how much I will get annoyed by red squiggly lines while coding. Let's leave it for now and see how I feel later.

Reply:
Personally I use the [prettier plugin](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) for vscode as well as format on save options, so as soon as I save the squiggly lines go away 😃